package.json

```diff
@@ -6,8 +6,8 @@
     "ecosystem:aio-app-builder-templates"
   ],
   "dependencies": {
-    "@adobe/generator-add-events-generic": "^0.1.3",
-    "@adobe/generator-app-common-lib": "0.4.2",
+    "@adobe/generator-add-events-generic": "^0.1.4",
+    "@adobe/generator-app-common-lib": "^0.4.5",
```

Comment:
the version was fixed before, should we remove the `^` as well now?

Reply:
i missed it last time. this is to specify that project should use the latest minor/patch version of the dependencies

Reply:
ok good! was just to double check, thanks :)